row_012_DB/row_012_DB_unsafe.py

```
@@ -0,0 +1,417 @@
+    connection.commit()
+    connection.close()
+
+
+def get_auth(user_id: int) -> []:
+    connection = sql.connect('userbase.db')
+    cursor = connection.cursor()
+
+    cursor.execute(f"""
+                    select FIO, branch from users
+                    where user_id = {user_id}
+                    """)
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)